src/components/inputs/radioButtons/rectangularRadioButton.js

```diff
@@ -2,13 +2,15 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import styled, { css } from 'styled-components'
 import ErrorMessage from 'SRC/components/inputs/ErrorMessage'
+import P from 'SRC/core/typography/P'
 
-const BaseRectangularRadioButton = ({children, className, value, error, ...props}) => {
+const BaseRectangularRadioButton = ({buttonLabel, helperText, className, value, error, ...props}) => {
   return (
     <div className={className}>
       <input id={value} type='radio' value={value} {...props} />
       <label htmlFor={value}>
-        {children}
```

Comment:
I think using `children` might give us a bit more flexibility than using `P` tags. For example, if we wanted to display an image or an icon inside the button. What do you think?

Reply:
Sounds great!